src/Workspaces/Core/Desktop/Workspace/Host/TemporaryStorage/TemporaryStorageServiceFactory.cs

```diff
@@ -30,8 +31,58 @@ public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
         /// </summary>
         internal class TemporaryStorageService : ITemporaryStorageService2
         {
+            /// <summary>
+            /// The maximum size of a single storage unit in a memory mapped file which is shared with other storage
+            /// units.
+            /// </summary>
+            private const long SingleFileThreshold = 128 * 1024;
+
+            /// <summary>
+            /// The size of a memory mapped file created to store multiple temporary objects.
+            /// </summary>
+            private const long MultiFileBlockSize = SingleFileThreshold * 32;
```

Comment:
Say exactly that.  "These values were picked because they seemed reasonable and we didn't observe anything negative when we tested.  If necessary they can be changed based on data that indicates otherwise."